indefinitepagerindicator/src/main/java/com/rbrooks/indefinitepagerindicator/IndefinitePagerIndicator.kt

```diff
@@ -4,17 +4,20 @@ import android.content.Context
 import android.content.res.Resources
 import android.graphics.Canvas
 import android.graphics.Paint
-import android.support.annotation.ColorInt
-import android.support.v4.content.ContextCompat
-import android.support.v4.view.ViewCompat
-import android.support.v4.view.ViewPager
-import android.support.v7.widget.LinearLayoutManager
-import android.support.v7.widget.RecyclerView
 import android.util.AttributeSet
 import android.util.DisplayMetrics
 import android.view.View
 import android.view.animation.DecelerateInterpolator
-
+import androidx.annotation.ColorInt
+import androidx.annotation.Keep
+import androidx.core.content.ContextCompat
+import androidx.core.view.ViewCompat
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import androidx.viewpager.widget.ViewPager
+import androidx.viewpager2.widget.ViewPager2
+
+@Keep
```

Comment:
why are you adding the @keep annotation?

Reply:
I enabled minification, so without the `@Keep` this class will be obfuscated

Reply:
I don't really see a reason why we should obfuscate this. It's just 1 class and it's OSS. We could even remove the build types from the build.gradle file. What do you think?

Reply:
Obfuscation is indeed not needed, but R8 is also able to shrink and optimize code, which is always welcome IMO

Reply:
I totally agree with you that the help that R8 gives via code optimization is great, but we don't use reflection and we don't have proguard rules in place. It's literally 1 class that won't be really affected by it. 
So how about we remove the @Keep from this PR and we can move forward?